packages/auth/test/integration/flows/totp.test.ts

```diff
@@ -55,9 +55,10 @@ describe(' Integration tests: Mfa enrollement using totp', () => {
   beforeEach(async () => {
     emulatorUrl = getEmulatorUrl();
     if (!emulatorUrl) {
-      mfaUser = null;
       auth = getTestInstance();
       displayName = 'totp-integration-test';
+      const cr = await signInWithEmailAndPassword(auth, email, password);
```

Comment:
Let's move changes in this file to a separate CL (if needed)

Reply:
This has been removed completely. Sorry about the confusion! :)